### PR TITLE
Fix ordering of RequestResponses

### PR DESCRIPTION
**Pull request by Jammjammjamm**
RequestResponses were being shown out of order due to insufficient precision in how timestamps are stored in the database. This branch adds a new field to store their order, as shown in the screenshots below (check the order from bottom to top).

Before:
Screen Shot 2020-06-10 at 12 52 15 PM

After:
Screen Shot 2020-06-10 at 2 45 12 PM

**Submitter:**
- [x] This pull request describes why these changes were made
- [x] Internal ticket links to this PR
- [x] Internal ticket is properly labeled (Community/Program)
- [x] Internal ticket has a justification for its Community/Program label
- [x] Code diff has been reviewed for extraneous/missing code
- n/a Tests are included and test edge cases
- [x] Tests/code quality metrics have been run locally and pass

**Reviewer 1:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

**Comment by okeefm** [comment]
To anyone testing this: if you get a `DataObjects::SyntaxError: Cannot add a PRIMARY KEY column`, the solution is to remove the `*.db` files in `<inferno root>/data/`.

Side note: we'll want to figure out how to do this automatically (or tell people to do this) if we merge this, because otherwise everyone's Inferno installs will stop booting when they upgrade to this commit.

File: lib/app/models/request_response.rb [hunk 1]
```diff
@@ -14,6 +14,7 @@ class RequestResponse
       property :response_body, Text
       property :direction, String
       property :instance_id, String
+      property :sequence, Serial, unique_index: true
```
Comment (radamson) [inline lib/app/models/request_response.rb:17]: No biggie, but I think something like `request_order` would be a better name.  "Sequence" already has a different meaning in inferno and when I first saw this while inspecting the database my mind was expecting something different.

**Comment by radamson** [comment]
Tested it out and it worked well!  Happy to have this merged as is or implement the name change if others agree it is a good idea.

Screen Shot 2020-06-25 at 7 35 19 AM

File: lib/app/models/request_response.rb (hunk 1 above)
Comment (okeefm) [reply lib/app/models/request_response.rb:17]: Yeah, `request_order` or `request_index` might make more sense for this, given that Sequence is already taken.

Comment (arscan) [reply lib/app/models/request_response.rb]: I third this suggestion, and updated the column.  Could some verify my update real quick?

**Comment by arscan** [comment]
Our plan regarding the DB change is to have users manually move or delete the existing database file.  This is a headache I wanted to avoid, but adding the complexity of migration scripts is risky without a good amount of thought.  And we need this bug (wrong ordered responses) fixed ASAP because it is misleading.

File: lib/app/models/request_response.rb (hunk 1 above)
Comment (radamson) [reply lib/app/models/request_response.rb]: Tested out the change and it worked as expected 👍
